builtin/game/fine_pointed.lua

```diff
@@ -0,0 +1,35 @@
+-- Returns the exact coordinate of a pointed surface
+
+local function fine_pointed(pos, camera_pos, pos_off, look_dir)
```

Comment:
'camera_pos' is possibly confusing as this is actually the feet-position of a player, 1.625 is added to the Y component later in the code.